public/index.html

```diff
@@ -1,102 +1,138 @@
 <!DOCTYPE html>
 <html lang="en" prefix="og: http://ogp.me/ns#">
+  <head>
```

Comment:
Most of these changes are due to prettier formatting. The only update was combining the two google font links and updating them to use the latest google fonts api.